core-bundle/assets/scripts/core.js

```diff
@@ -554,6 +555,7 @@ window.Backend =
 			'hideFooter': true,
 			'draggable': false,
 			'overlayOpacity': .7,
+			'overlayClick': false,
```

Comment:
we should not add it here, since `openModalImage` is just an image preview in the file tree, where closing with click on modal seems perfectly fine with me.

Reply:
Ah, that's what this one is for. Have been looking for where this has been used and the only occurences were Contao 3.x forks and isotope :D

Reply:
This might have been removed completely in Contao 4 and 5 but may have never been deprecated I guess?

Reply:
at least Isotope is using it 😬